magento/pwa-studio
# [PWA-1512]: Check if email is associated with account in guest checkout

## Description

New feature that actively checks if the email typed by a guest customer is associated with an existing account and if so, renders a toast message to suggest a signIn.

 
## Related Issue







Closes PWA-1512

## Acceptance




1. User enters email address at checkout. If there is an account associated with the email, user sees a message about that and has the option to enter the password.
2. User has an option to recover a password
3. User enters valid password, logs-in and continues checkout as a customer, not a guest. If user does not provide a valid password, they can't login and will have to proceed as a guest with the email provided

### Verification Stakeholders



### Specification




## Verification Steps







#### Test scenario(s) for direct fix/feature




1. Go to Venia home page
2. Create an account
3. Sign out
2. Add a product to cart
3. Go to checkout
4. Type the email that was used to create an account on Shipping Information
5. Click on next field
6.  A toast message should appear suggesting a sign in
7. Click in Yes, sign in
8. A sign in form should appear autofilled with the previously typed email 

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist
